src/cmd/root.go

```diff
@@ -80,6 +81,7 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&message.NoProgress, "no-progress", v.GetBool(V_NO_PROGRESS), lang.RootCmdFlagNoProgress)
 	rootCmd.PersistentFlags().StringVar(&config.CommonOptions.CachePath, "zarf-cache", v.GetString(V_ZARF_CACHE), lang.RootCmdFlagCachePath)
 	rootCmd.PersistentFlags().StringVar(&config.CommonOptions.TempDirectory, "tmpdir", v.GetString(V_TMP_DIR), lang.RootCmdFlagTempDir)
+	rootCmd.PersistentFlags().BoolVar(&config.CommonOptions.Insecure, "insecure", v.GetBool(V_INSECURE), lang.RootCmdFlagInseure)
```

Comment:
Can we add a `-k` alias like we had discussed earlier in the week to signal to folks this more closely follows `curl`s paradigms? 

```suggestion
	rootCmd.PersistentFlags().BoolVarP(&config.CommonOptions.Insecure, "insecure", "k", v.GetBool(V_INSECURE), lang.RootCmdFlagInseure)
```